packages/gp2-components/src/templates/DashboardPageBody.tsx

```diff
@@ -93,11 +93,11 @@ const DashboardPageBody: React.FC<DashboardPageBodyProps> = ({
         <div css={contentCardsStyles}>
           <InfoCard
             icon={<LabIcon color="#00202C" size={40} />}
-            title="Samples Completed"
+            title="Samples shared w/AMP PD"
```

Comment:
where is this copy? in figma, this didn't change.

Reply:
It's in the spreadsheet attached to the ticket. It's been changed to 'Samples Processed & Shared' now which I've just updated it to